### PR TITLE
Resolve search entries

### LINKED ISSUE #33: Feature: Search entries
**Describe the solution you'd like**
As a user, I want to be able to search my entries over my list entries not deleted, using a input text.

- [ ] The entries to show have to include the input text either by the title or by the content.
- [ ] All entries will show when the input text is empty

### PR DESCRIPTION
**Pull request by paulotijero**
Resolve #33

### CODE REVIEW

File: route.rb [hunk 1]
```diff
@@ -87,4 +81,12 @@
 get "/restore" do
   update_delete_data(params["id"] , 0)
   redirect "/trash"
+end
+
+post "/search" do
```
Comment (liannivin) [inline route.rb]: Good!

File: route.rb [hunk 2]
```diff
@@ -87,4 +81,12 @@
...
+  @url = "/search"
+  @data = search(params)
+  erb :list
+  erb :list_entry
+  erb :entry
```
Comment (liannivin) [inline route.rb]: Good

File: views/list.erb [hunk 3]
```diff
@@ -17,18 +17,25 @@
         </a>
       </div>
     </div>
-    <% if @url == ( "/" || "/edit" ) %>
+    <% if @url == "/" || @url == "/edit" || @url == "/search" %>
```
Comment (liannivin) [inline views/list.erb]: Could you explain the utility of this change?

File: views/list.erb [hunk 4]
```diff
@@ -17,18 +17,25 @@
...
-            <button type="button" class="btn btn-default btn-sm" title="Agregar">
-              <span
-                    class="glyphicon glyphicon-plus-sign"
-                    aria-hidden="true"
-              ></span>
-            </button>
-          </a>
+          <form action="/search" method="post">
+          <div class="input-group">
+            <input type="text" name="search" class="form-control" placeholder="Buscar..." value="<%= params["search"] %>">
```
Comment (Linzeur) [inline views/list.erb]: I like that it will always show the value of input text!!

**Pull request merged**